ops/cli.py

```diff
@@ -248,30 +261,35 @@ def print_deployment_success(config: WflInstanceConfig) -> None:
 
 def make_git_tag(config: WflInstanceConfig) -> None:
     info("=>  Tagging current commit with version")
-    shell(f"git tag -a v{config.version} -m 'Created by cli.py {config.command}'")
-    shell(f"git push origin v{config.version}")
+    shell(f"git tag -a v{config.version} -m 'Created by cli.py {config.command}'", cwd=config.wfl_root_folder)
+    shell(f"git push origin v{config.version}", cwd=config.wfl_root_folder)
     success(f"Tag 'v{config.version}' created and pushed")
```

Comment:
I really don't think this script should be doing this.
It should be using a release, not creating one.

Reply:
The part that uses a release is over in the Jenkins job that actually deploys WFL. Just like you mentioned in your comment, we should tag WFL in this job and this is the part that does that